example/publishAndFetchPublic.html

```diff
@@ -0,0 +1,66 @@
+<html>
+
+<head>
+  <title>MAM Example Publish and Fetch</title>
+</head>
+
+<body>
+  <div id="output"></div>
+  <script src="../lib/mam.web.js"></script>
+  <script>
+    (async function() {
+      const mode = 'public'
+      const provider = 'https://nodes.devnet.iota.org'
+
+      const mamExplorerLink = `https://mam-explorer.firebaseapp.com/?provider=${encodeURIComponent(provider)}&mode=${mode}&root=`
+
+      const outputHtml = document.querySelector("#output");
+
+      // Initialise MAM State
+      let mamState = Mam.init(provider)
+
+      // Publish to tangle
+      const publish = async trytes => {
+        // Create MAM Payload - STRING OF TRYTES
+        const message = Mam.create(mamState, trytes)
+
+        // Save new mamState
+        mamState = message.state
+
+        // Attach the payload
+        await Mam.attach(message.payload, message.address, 3, 9)
+
+        outputHtml.innerHTML += `Published: ${trytes}<br/>`;
+        return message.root
+      }
+
+      const publishAll = async () => {
+        const root = await publish('ALICE')
+
+        await publish('BOB')
```

Comment:
It will be almost never a use case, where a plain string is published instead of an object. 
If you want to publish an object like `{
    message: 'Message from Alice',
    timestamp: (new Date()).toLocaleString()
 }`, you will need to add `asciiToTrytes`, `trytesToAscii` functions

Reply:
As we don't have a module loader in the simple html example we can't use iota.js, so no way to easily import asciiToTrytes and trytesToAscii

Reply:
as workaround you can re-define these functions and add them into the file or as another importable module
```
const TRYTE_ALPHABET = '9ABCDEFGHIJKLMNOPQRSTUVWXYZ';

const asciiToTrytes = (input) => {
  if (!/^[\x00-\x7F]*$/.test(input)) {
    throw new Error('Invalid ascii charactes.');
  }
  let trytes = '';
  for (let i = 0; i < input.length; i++) {
    var dec = input[i].charCodeAt(0);
    trytes += TRYTE_ALPHABET[dec % 27];
    trytes += TRYTE_ALPHABET[(dec - dec % 27) / 27];
  }
  return trytes;
};

const trytesToAscii = (trytes) => {
  if (typeof trytes !== 'string' || !new RegExp("^[9A-Z]{1,}$").test(trytes)) {
    throw new Error('Invalid trytes');
  }
  if (trytes.length % 2) {
    throw new Error('Invalid trytes length. Expected trytes of even length.');
  }
  let ascii = '';
  for (let i = 0; i < trytes.length; i += 2) {
    ascii += String.fromCharCode(TRYTE_ALPHABET.indexOf(trytes[i]) + TRYTE_ALPHABET.indexOf(trytes[i + 1]) * 27);
  }
  return ascii;
};
```

Reply:
Example updated